src/core/annotation.js

```diff
@@ -332,6 +337,30 @@ class Annotation {
       this._streams.push(this.appearance);
     }
 
+    let actions = null;
+    let fieldName = null;
+    let kidsId = null;
```

Comment:
Given that this is an array of IDs, I'd expect a name like `kidsIds` or maybe `kidIds`, but at least IDs in plural.